src/definitions.php

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>EKGViewer - Definitions</title>
+    <link href="./css/reset.css" rel="stylesheet" />
+    <link href="./templates/css/page-template.css" rel="stylesheet" />
+    <link href="./css/definitions.css" rel="stylesheet" />
+</head>
```

Comment:
Each page has uses the reset css, the page-template css, and its own css files. The reset css removes the styling added by browsers (makes everything consistent). The page-template css allows the header and footers to be styled correctly.